src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -8831,5 +8831,26 @@ public async Task TestCollectionExpression_Start_Typeless()
 "var x = $$[1, 2]";
             await TestAsync(source);
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/71638")]
+        public async Task TestAnonymousType()
+        {
+            var markup = """
+                _ = new
+                {
+                    @string = ""
+                }.$$@string;
+                """;
+            var description = $"string 'a.@string {{ get; }}";
```

Comment:
💡 It would be good to include a property that doesn't need the `@` as well. Not sure if there is already a test in this file for such a case.

Reply:
@sharwell I think [this one](https://github.com/dotnet/roslyn/blob/7959fedb88c5dab6f2c186be4a4f3fa8e6211359/src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs#L4352) should be good to cover this case?

Reply:
Yep thanks!